lms/services/h_api.py

```diff
@@ -81,17 +80,15 @@ def _api_request(self, method, path, body=None, headers=None):
             request_args["data"] = body
 
         try:
-            response = requests.request(
+            response = self._http_service.request(
                 method=method,
                 url=self._base_url + path.lstrip("/"),
                 auth=self._http_auth,
                 timeout=10,
```

Comment:
timeout=10 is the default, unless clients have a good reason to overwrite it I'd leave it out

Reply:
Yeah, I think maybe make `10` the default (it's currently `9`) then remove it here

Reply:
Done: https://github.com/hypothesis/lms/pull/2711/commits/d54c0796f3b5b15119b7b59ea00dadff0ce54f1e